atom/browser/browser_mac.mm

```diff
@@ -202,29 +202,119 @@
   return prevent_default;
 }
 
+static LSSharedFileListItemRef GetItemFromLoginItems(
```

Comment:
This looks very similar to https://chromium.googlesource.com/chromium/src/+/69.0.3497.106/base/mac/mac_util.mm#84, any chance we can re-use the code in `base::mac`?

Reply:
Unfortunate the crux of the fix is in `CheckLoginItemStatus` which requires a call to `GetItemFromLoginItems`; Chromium's version isn't public so we'd need to patch in order to use it